apps/etterlatte-fordeler/src/main/kotlin/fordeler/FordelerService.kt

```diff
@@ -52,6 +56,13 @@ class FordelerService(
                     Vedtaksloesning.PESYS -> IkkeGyldigForBehandling(it.kriterier)
                 }
             }
+        } catch (e: FamilieRelasjonManglerIdent) {
```

Comment:
Skjønte ikke helt hvordan denne fanges opp? ser ut som den tilsvarende exception throwes i `pdltjenester` men det propagerar vel ikke ut hit?

Reply:
Her må vi nok via en statuskode for å kunne gjøre dette siden dette er to ulike apper ja

Reply:
ah, det er nok sant. Jeg tenkte for raskt i min hacky'e løsning :D